source/requirements.rst

```diff
@@ -9,9 +9,12 @@ PHP 5.5.9 or later is required.
 
 PHP dependencies are handled with `Composer`_. Composer should be installed globally. The Symfony framework has additional requirements for PHP.
 
+.. note:: Some php extensions may not be activated. One such extension that is used in DHIL Symfony projects is the `Normalizer`_ class. To enable it, find the php.ini file on your disk, open it, and remove the comment sign ";" in front of it.
+
```

Comment:
Normalizer is one class inside the intl extension. You need to enable the intl extension, not the normalizer class.

Reply:
fixed!